lib/coverband/reporters/base.rb

```diff
@@ -20,25 +21,14 @@ def report(store, _options = {})
 
         protected
 
-        def root_paths
-          roots = Coverband.configuration.root_paths
-          roots += Coverband.configuration.gem_paths if Coverband.configuration.track_gems
-          roots << "#{current_root}/"
-          roots
-        end
-
-        def current_root
-          File.expand_path(Coverband.configuration.root)
-        end
-
         def fix_file_names(report_hash, roots)
           if Coverband.configuration.verbose
             Coverband.configuration.logger.info "fixing root: #{roots.join(', ')}"
           end
 
           # normalize names across servers
           report_hash.each_with_object({}) do |(key, vals), fixed_report|
-            filename = filename_from_key(key, roots)
+            filename = relative_path_to_full(key, roots)
```

Comment:
I'm surprised we have to convert back from relative to absolute? I pictured this only going one way.

Reply:
all the web reporting / HTML expects a full path. Part of this is legacy as SimpleCov always expected full path. So further refactorings are likely possible but it would touch most if not all the SimpleCov files so it is a fairly major change.

Reply:
Also not true for Gems which aren't relative paths

Reply:
I made a comment below, because I'm not sure that the reverse is even technically possible in all cases, especailly with regex... 

But besides that, I'm not sure if I understand why we care for the path to be "real" real, rather than normalized / canonicalized. Does it mean SimpleCov won't be able to open our file unless we give it the full path? (sorry, no clue about SimpleCov or where it's used etc)